provisioner/terraform/serve_internal_test.go

```diff
@@ -39,7 +39,7 @@ func Test_absoluteBinaryPath(t *testing.T) {
 		{
 			name:             "TestNewVersion",
 			args:             args{ctx: context.Background()},
-			terraformVersion: "1.3.0",
+			terraformVersion: "1.3.6",
```

Comment:
We should probably not bump this and just change it to `99.999.999` so that it's always "new".

```suggestion
			terraformVersion: "99.999.999",
```